doc/sphinx/changes.rst

```diff
@@ -662,11 +662,16 @@ Infrastructure and dependencies
   by Emilio Jesus Gallego Arias, review by Vincent Laporte, Guillaume
   Melquiond, Enrico Tassi, and Théo Zimmerman).
 - **Changed:**
-  Coq's OCaml parts and tools [coq-core] are now built using Dune.
+  Coq's OCaml parts and tools [``coq-core``] are now built using Dune.
   The main user-facing change is that Dune >= 2.5 is now required to
-  build Coq, most recent Dune is usually recommended.
+  build Coq, most recent Dune is usually recommended. This was a large and
```

Comment:
```suggestion
  build Coq.  We usually recommend using the most recent version of Dune. This was a large and
```

Can you say when would it be appropriate to use an older version of Dune?

Reply:
There might be reasons why people want to use an older version, e.g. because they use their system package manager to install it. But the latest version is always recommended.